nginx/openflights-dev

```diff
@@ -0,0 +1,39 @@
+server {
+    root /var/www/openflights;
+
+    index index.html index.php;
+
+    location / {
+        ssi on;
+        if (-e $request_filename.php){
+            rewrite ^(.*)$ /$1.php;
+        }
+        try_files $uri $uri.html $uri/ =404;
+    }
+
+    location ~ \.php$ {
+        include snippets/fastcgi-php.conf;
+        fastcgi_pass unix:/var/run/php/php7.4-fpm.sock;
```

Comment:
This isn't going to work for PHP 8.1 of course

Reply:
Right, forgot I have a local change for this in the PHP8 container.

I could symlink these to an unversioned name or update `/etc/php/fpm/pool.d/www.conf` to remove the version from the name.